### ISSUE TITLE
ember-href-to last version throws ember warnings for some deprications

### ISSUE THREAD

**Issue by usamahamed** [intercom/ember-href-to#146]
I have the last version of ember-href-to `4.1.0` and it throws a deprecation warning with this id ` ember.built-in-components.import` 

```
testem.js:967 DEPRECATION: Using Ember.LinkComponent or importing from 'LinkComponent' has been deprecated, install the @ember/legacy-built-in-components addon and use import { LinkComponent } from '@ember/legacy-built-in-components'; instead [deprecation id: ember.built-in-components.import] See Ember.js - Deprecations  for more details.
```
And this when I debug the depreciation cause 

![image](https://user-images.githubusercontent.com/23267284/137459661-c9776562-e81c-4d63-a395-501994c632b6.png)

**Comment by SergeAstapov** [comment]
This is now hard error under ermber-source v4.0

**Comment by patocallaghan** [comment]
Sorry, I completely missed this. I'll try to address this early next week if no one else gets to it first.

**Comment by cibernox** [comment]
Hard error in `ember-release` now indeed.

**Comment by patocallaghan** [comment]
Sorry for the delay. This has been fixed in [v5.0.0](https://github.com/intercom/ember-href-to/releases/tag/v5.0.0)
